airflow/models/serialized_dag.py

```diff
@@ -350,7 +347,7 @@ def get_max_last_updated_datetime(cls, session: Session = NEW_SESSION) -> dateti
 
         :param session: ORM Session
         """
-        return session.query(func.max(cls.last_updated)).scalar()
+        return session.scalar(func.max(cls.last_updated))
```

Comment:
We might need a select here i.e: `session.scalar(select(func.max(cls.last_updated)))` but if it works, then no issue

Reply:
works without select, however changed it as per your suggestion